scripts/build_docker.py

```diff
@@ -149,9 +149,9 @@ def get_docker_command(
         build_target = "dev"
     elif build_preset == "lean":
         build_target = "lean"
-    elif build_preset == "py310":
+    elif build_preset == "py311":
```

Comment:
optional, but we could add a `py39` preset here while we support 3.9 (until we remove it in setup.py)